cmd/create/cluster/cmd.go

```diff
@@ -545,6 +546,13 @@ func initFlags(cmd *cobra.Command) {
 			"without exposing your traffic to the public internet.",
 	)
 
+	flags.BoolVar(
+		&args.privateIngress,
+		"private-ingress",
```

Comment:
Just considering what could be used here `--default-ingress-private` to follow the other default-ingress args